src/snowflake/connector/connection.py

```diff
@@ -607,8 +613,13 @@ def __open_connection(self):
                 PARAMETER_CLIENT_STORE_TEMPORARY_CREDENTIAL] = \
                     self._client_store_temporary_credential if IS_LINUX else True
 
+        if self._authenticator == USR_PWD_MFA_AUTHENTICATOR:
+            self._session_parameters[
+                PARAMETER_CLIENT_REQUEST_MFA_TOKEN] = \
+                    self._client_request_mfa_token if IS_LINUX else True
```

Comment:
This feature is only open for Mac & Windows. Linux support is used for internal testing. And may become public in the future.